src/main/java/de/fau/amos4/util/EmailSender.java

```diff
@@ -0,0 +1,56 @@
+package de.fau.amos4.util;
+import java.util.Properties;
+
+import javax.mail.*;
+import javax.mail.internet.*;
+
+public class EmailSender {
+	// Implements basic email sending functionality
+	
+	// The "from" Email address in the sent mesasges
+	// TO DO: Configuration settings for Sender, MailServerHost and Credentials
+	private String SenderEmail = "PersonalFragebogen2.0@gmail.com";	
```

Comment:
You can put this configuration data into the *.yml files. This way we can easily change it depending on the application deployment environment. You can load them with the @Value annotation.